ansible/roles/openshift_sso_app_zabbix/meta/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+galaxy_info:
+  author: OpenShift Ops
+  description: Openshift Operations SSO Zabbix Role
+  company: Red Hat, Inc
+  license: ASL 2.0
+  min_ansible_version: 2.0
+  platforms:
+  - name: EL
+    versions:
+    - 7
+dependencies:
+- role: tools_roles/lib_openshift_3.2
+- role: tools_roles/os_zabbix_cluster_stats
```

Comment:
this seems like a mistake. that's probably what your other comment meant about putting it in the tech profile, and I agree, but take care that you call your new role, and not os_zabbix_cluster_stats